container/README.md

```diff
@@ -63,41 +104,54 @@ npx proteinpaint-container server
 
 To run a fully customized server, follow the docker run commands in `run.sh`.
 
-## Development
-
-### Installing from tarballs
+### Build from Source
 
 The Dockerfile may optionally use the various proteinpaint-* packages as packed from source code.
 
 ```bash
-# will pack workspaces and replace each package.json's dependency versions
-# with the tarball location as copied into the Docker build
+# OPTIONAL: pack workspaces and replace each package.json's 
+# dependency versions with the tarball location as copied into the Docker build
+# if this script is not used, then published packages will be used in the Docker build
 ./pack.sh
 
 # will run docker build
 ./build.sh
-```
 
-### Testing and Debugging
+TAG=latest # can change to a version number like 2.11.2
+IMAGE_NAME=full:$TAG # may use server:$TAG for server-only image
+docker pull ghcr.io/stjude/proteinpaint/$IMAGE_NAME
```

Comment:
We would have to agree on the docker image name format.
It can be also like:

`ghcr.io/stjude/ppfull-$TAG`

Like here:
https://github.com/stjude/proteinpaint/pkgs/container/ppfull

@robinpaul85 @siosonel  which one is better? WDYT?